cloudsync/sync/state.py

```diff
@@ -141,7 +141,7 @@ def needs_sync(self):
         return self.changed and self.oid and (
                self.hash != self.sync_hash or
                self.parent.paths_differ(self.side) or
-               self.exists in (TRASHED, LIKELY_TRASHED))
+               self.exists in (TRASHED, LIKELY_TRASHED))  # do we want to add MISSING here?
```

Comment:
@earthman1 - I assume you were considering doing this instead of doing a special "missing-not-finished" check in the manager sync loop (which is the current impl) - can i remove this comment now, or are there other reasons to consider this?

Reply:
The comment was there to remind me to consider adding MISSING at that spot. If your tests are passing without it there, then no need to add it